packages/core/src/components/advanced-select/advanced-select.scss

```diff
@@ -0,0 +1,186 @@
+@use "sass:math";
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/global/mixins/set-colors.mixin" as set-colors;
+@use "~dso-toolkit/src/components/advanced-select";
+
+:host {
+  display: block;
+}
+
+@include utilities.box-sizing();
+
+.active-option {
+  display: flex;
+  width: 100%;
```

Comment:
Als we nieuwe CSS schrijven, maken we uitsluitend gebruik van logical properties (https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values). In dit geval zou dat `inline-size` moeten zijn ipv `width`. Ik zal een remark zetten op iedere plek waar een logical property moet komen. Ik zal je daar de eerstvolgende keer als ik in Apeldoorn ben, wel even over bijpraten ;)

Reply:
Ik wist niet eens dat dat een ding was. Compleet nieuw voor mij dit 😮 . Maar heb het aangepast! 😄